### PR TITLE
add new button component

### PR DESCRIPTION
**Pull request by vanpertsch**
Before | After
-- | --
Screenshot 2022-03-03 at 10 31 52 | Screenshot 2022-03-03 at 10 07 30
Screenshot 2022-03-03 at 10 32 33 | Screenshot 2022-03-03 at 10 07 45

### CODE REVIEW

File: apps/settings/src/components/PersonalInfo/shared/HeaderBar.vue [hunk 1]
```diff
@@ -35,25 +35,31 @@
 		</template>
 
 		<template v-if="isEditable && isMultiValueSupported">
-			<AddButton class="add-button"
-				:disabled="!isValidSection"
-				@click.stop.prevent="onAddAdditional" />
+			<Button :disabled="!isValidSection"
+				:aria-label="t('settings', 'Add email')"
```
Comment (Pytal) [inline apps/settings/src/components/PersonalInfo/shared/HeaderBar.vue:39]:
```suggestion
				:aria-label="t('settings', 'Add additional email')"
```

**Review by Pytal — APPROVED** [review]
Other than the quick aria label fix :)

**Comment by vanpertsch** [comment]
@jancborchardt do you agree with the change of the button here?

**Comment by jancborchardt** [comment]
The tertiary button style would look better here, and it's also closer to how the button was before. :)

File: apps/settings/src/components/PersonalInfo/shared/HeaderBar.vue [hunk 2]
```diff
@@ -137,7 +143,8 @@ export default {
 		margin: -12px 0 0 8px;
 	}
 
-	.add-button {
+	.button-vue  {
 		margin: -12px 0 0 auto !important;
+		height: 44px;
```
Comment (artonge) [inline apps/settings/src/components/PersonalInfo/shared/HeaderBar.vue:148]: Is it necessary to set the height? Is the default not working?

Comment (vanpertsch) [reply apps/settings/src/components/PersonalInfo/shared/HeaderBar.vue]: you are right. Changed the negativ margin to -6px and the default button min-height is working. thx

**Review by artonge — APPROVED** [review]

**Review by marcoambrosini — CHANGES_REQUESTED** [review]
We shouldn't use buttons type "secondary" for now, see [frontenders chat](https://cloud.nextcloud.com/call/dxbbx4jy)

**Review by marcoambrosini — APPROVED** [review]
uups sorry I saw the screenshot and thought you were using it

**Pull request merged**